### PR TITLE
Update README files

### PR DESCRIPTION
**Pull request by praneetloke**
### Proposed changes

Follow-up from https://github.com/pulumi/docs/pull/6788. Removing some old information which was already not accurate based on how things were before the aforementioned PR was merged.

### Related issues (optional)

pulumi/registry#237

### CODE REVIEW

File: tools/resourcedocsgen/README.md [hunk 1]
```diff
@@ -1,15 +1,68 @@
 # Resource Docs generator driver
 
-This tool generates the provider schema using `tfgen`, then calls the docs generator tool in `pulumi/pulumi` to use the schema to generate docs.
+This tool calls the docs generator tool in `pulumi/pulumi` which uses the Pulumi schema for a package to generate API (resource) docs.
 
-## Generating docs
+## Installation
 
-Run `make resource_docs`.
+You can install the `resorucedocsgen` tool just like any other Go-based CLI tool:
```
Comment (lblackstone) [inline tools/resourcedocsgen/README.md:7]:
```suggestion
You can install the `resourcedocsgen` tool just like any other Go-based CLI tool:
```

Comment (praneetloke) [reply tools/resourcedocsgen/README.md:7]:
```suggestion
You can install the `resourcedocsgen` tool just like any other Go-based CLI tool:
```

Comment (praneetloke) [reply tools/resourcedocsgen/README.md:7]: Thanks for catching that, Levi! For some reason GH won't let me commit the suggestion from the UI here. I'll push an update.

**Comment by praneetloke** [comment]
Ping

**Review by cnunciato — APPROVED** [review]

**Pull request merged**